specification/authorization/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the Authorization API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2018-09-01-preview

```

Comment:
Did you intend to lower the version in the basic information section?

Reply:
Yes. The Deny Assignment APIs were added some time back and so they have the api version 2018-07-01-preview. Recently, some updates were made to existing Role Assignment APIs that were part of the newer 2018-09-01-preview. Problem is that the newer API version is not deployed yet and so we need to go with the 2018-07-01-preview for now. However, as you the specification/authorization/resource-manager/readme.md file has been updated to include the new swagger spec in both tags (2018-07-01-preview and 2018-09-01-preview) and so the sdk generated by both tags will include the new 2018-07-01-preview swagger spec. When the new 2018-09-01-preview gets deployed then we will update the default tag to 2018-09-01-preview. That way customers can use either sdk and will still get the same new set of APIs for deny assignments.